code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm

```diff
@@ -464,3 +545,53 @@
 	if(!origin_turf)
 		return
 	origin_turf.assume_air(remove_mixture)
+
+/obj/machinery/atmospherics/components/unary/hypertorus/core/proc/check_spill(delta_time)
+	var/obj/machinery/atmospherics/components/unary/hypertorus/cracked_part = check_cracked_parts()
+	if (cracked_part)
+		// We have an existing crack
+		var/leak_rate
+		if (moderator_internal.return_pressure() < HYPERTORUS_MEDIUM_SPILL_PRESSURE)
+			// Not high pressure, but can still leak
+			if (!prob(HYPERTORUS_WEAK_SPILL_CHANCE))
+				return
```

Comment:
Is this consistent with the old behavior? I remember looking at this block of code and thinking it would be a pain to do what you're doing right here so early returns don't fuck it